libs/ReinforcerKit/source/ReinforcerKit.cpp

```diff
@@ -0,0 +1,89 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "ReinforcerKit.h"
+
+using namespace leka;
+using namespace std::chrono;
+
+static constexpr auto reinforcer_duration = 2700ms;
+
+void ReinforcerKit::setDefaultReinforcer(const Reinforcer &reinforcer)
+{
+	_default_reinforcer = reinforcer;
+}
+
+void ReinforcerKit::run(const Reinforcer &reinforcer)
+{
+	switch (reinforcer) {
```

Comment:
Suggestion un peu barbare, avoir un run qui prend en paramètre un Reinforcer en pointeur et s'il est nul de pouvoir lancer le renforçateur par défaut.

```suggestion
void ReinforcerKit::run(const Reinforcer *reinforcer = nullptr)
{
	auto selected_reinforcer = *reinforcer == nullptr ? _default_reinforcer : *reinforcer
	switch (selected_reinforcer) {
```

---

Ca permettrait de l'extérieur de simplifier encore plus les choses en ayant uniquement que `setDefaultReinforcer` 
 et `run` en fonctions d'appels de ReinforcerKit plutôt que de se dire "si je ne sais pas quel renforçateur appeler, il faut appeler une autre fonction"